docs/docs/frontend-protocol.md

```diff
@@ -73,7 +73,7 @@ note for `new_view`. Errors are not currently reported.
 
 `set_theme {"theme_name": "InspiredGitHub"}`
 
-Requests that core change the theme. If the change succeeds the client
+Requests that core changes the theme. If the change succeeds the client
```

Comment:
This was correct before, although I appreciate if it isn't very clear. 'Change' is the future tense, and 'changes' is the present; we're requesting that the core change the theme at some point in the (immediate) future.

It's probably more confusing because 'Requests' is present. I think this makes sense though; you could read it like "She asks that he turn down the music."